index.html

```diff
@@ -167,54 +274,84 @@ <h4>Frameworks</h4>
                         <li>Capibara</li>
                         <li>Selenium </li>
                     </ul>
-                    <div class="bg-img-2"></div>
+                    <div class="rectangle_19"></div>
+                </div>
+                <div class="about_me">
+                    <h3>About</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
                 </div>
-                <div class="card-3 experiece">
+            </div>
+            <div class="skill_dot_bt">
+                <div class="skills">
                     <h4>Skills</h4>
-                    <ul class="tool group">
+                    <ul class="group">
                         <li>Codekit</li>
                         <li>Codepen</li>
                         <li>GitHub</li>
                         <li>GitLAb</li>
                         <li>Terminal</li>
                     </ul>
+                    <div class="vector_1"></div>
+                </div>
+                <div class="dots_vec">
+                    <img src="images/image_dots_with_teal.png" alt="image_dots_with_teal">
+                </div>
+                <div class="about_me">
+                    <a class="cv" href="">
+                        <p class="mycv">Get my Cv </p>
+                        <img class="ic_download" src="images/Group-61.png" alt="img-group-61">
+
+                    </a>
                 </div>
             </div>
 
         </section>
         <footer class="footer">
-            <div class="icon-geometry"></div>
+            <div class="icon-geometry_2"></div>
             <div class="get_started">
                 <p class="get_started_header">Get started</p>
                 <p class="get_started_text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
 
             </div>
-            <form action="https://formspree.io/f/xknkvkgk" method="POST">
+            <form class="mobile_form" action="https://formspree.io/f/xknkvkgk" method="POST">
```

Comment:
- [ ] You should add more padding for your inputs in your form to center a little bit the next in its container, 
- [ ] It would be nice if you diminish the space between the `textarea` and your submit `button` to make it more similar to the Figma design you selected.

![image](https://user-images.githubusercontent.com/60164239/131208341-baf366df-b4fc-4306-96f1-97d34ee9aeb9.png)